data/src/main/java/com/microsoft/azure/kusto/data/exceptions/KustoServiceError.java

```diff
@@ -9,21 +9,46 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/*
+  This class represents an error that returned from the service
+ */
 public class KustoServiceError extends Exception {
-    private ArrayList<Exception> exceptions = null;
+    private ArrayList<Exception> exceptions;
 
-    public KustoServiceError(JSONArray exceptions) throws JSONException {
+    public KustoServiceError(JSONArray jsonExceptions, boolean isOneApi, String message) throws JSONException {
+        this(message);
         this.exceptions = new ArrayList<>();
```

Comment:
This line is unnecessary, as the other constructor is called which already does this.